blueprints/ember-cli-typescript/files/app/config/environment.d.ts

```diff
@@ -9,4 +9,7 @@ export default config;
  */
 declare namespace config {
   export var environment: any;
+  export var modulePrefix: String;
```

Comment:
Thanks for the heads up. Also brand new to ts.

https://github.com/emberwatch/ember-cli-typescript/pull/27